LICENSE_GEMS

```diff
@@ -0,0 +1,131 @@
+actionpack, 4.2.8, MIT
+actionview, 4.2.8, MIT
+activemodel, 4.2.8, MIT
+activerecord, 4.2.8, MIT
+activesupport, 4.2.8, MIT
+addressable, 2.5.1, "Apache 2.0"
+arel, 6.0.4, MIT
+arel-helpers, 2.3.0, unknown
+aruba, 0.14.2, MIT
+bcrypt, 3.1.11, MIT
+bindata, 2.3.5, ruby
+bit-struct, 0.15.0, ruby
+builder, 3.2.3, MIT
+bundler, 1.14.6, MIT
+capybara, 2.13.0, MIT
+childprocess, 0.5.9, MIT
+coderay, 1.1.1, MIT
+contracts, 0.15.0, "Simplified BSD"
+cucumber, 2.4.0, MIT
+cucumber-core, 1.5.0, MIT
+cucumber-rails, 1.4.5, MIT
+cucumber-wire, 0.0.1, MIT
+diff-lcs, 1.3, "MIT, Artistic-2.0, GPL-2.0+"
+docile, 1.1.5, MIT
+erubis, 2.7.0, MIT
+factory_girl, 4.8.0, MIT
+factory_girl_rails, 4.8.0, MIT
+faraday, 0.12.0.1, MIT
+ffi, 1.9.18, "New BSD"
+filesize, 0.1.1, MIT
+fivemat, 1.3.3, MIT
+gherkin, 4.1.1, MIT
+google-protobuf, 3.2.0.2, "New BSD"
+googleauth, 0.5.1, "Apache 2.0"
+grpc, 1.2.0, "New BSD"
+i18n, 0.8.1, MIT
+jsobfu, 0.4.2, "New BSD"
+json, 2.0.3, ruby
+jwt, 1.5.6, MIT
+little-plugger, 1.1.4, MIT
+logging, 2.2.0, MIT
+loofah, 2.0.3, MIT
+memoist, 0.15.0, MIT
+metasm, 1.0.3, LGPL
+metasploit-aggregator, 0.1.3, "New BSD"
+metasploit-concern, 2.0.3, "New BSD"
+metasploit-credential, 2.0.8, "New BSD"
+metasploit-framework, 4.14.9, "New BSD"
+metasploit-model, 2.0.3, "New BSD"
```

Comment:
I'm actually just fixing the license splat in our LICENSE file in all of the rex-* gems. It shouldn't be a big problem.